### DISCUSSION TITLE
Enable SMTP Authentification on port 25

### DISCUSSION THREAD

**Discussion by ullrimax** [docker-mailserver/docker-mailserver#3391] [Q&A]
In the 12.0.0 update, [SMTP Authentication on port 25 has been disabled](https://github.com/docker-mailserver/docker-mailserver/pull/3006).

Is there a way to enable it again ?

Thanks

**Comment by casperklein** [comment]
Put this in your [user-patches.sh](https://docker-mailserver.github.io/docker-mailserver/latest/faq/#how-to-adjust-settings-with-the-user-patchessh-script):

`postconf "smtpd_sasl_auth_enable = yes"`

**Comment by polarathene** [comment]
> Is there a way to enable it again ?

Would you be able to chime in the use-case instead of preferring ports 465 or 587 which are intended for mail submission?

There was a recent report related to this: https://github.com/docker-mailserver/docker-mailserver/issues/3386#issuecomment-1589145492

Another option available to you is to use the [`postfix-main.cf` override config](https://docker-mailserver.github.io/docker-mailserver/latest/config/advanced/override-defaults/postfix/).
